src/connectors/dynamic-widgets/connectDynamicWidgets.ts

```diff
@@ -170,8 +170,21 @@ const connectDynamicWidgets: DynamicWidgetsConnector =
           const attributesToRender =
             results.renderingContent?.facetOrdering?.facets?.order ?? [];
 
+          const transformedAttributesToRender = transformItems(
+            attributesToRender,
+            { results }
+          );
```

Comment:
I'm ok with the existing suggestion as well

```suggestion
          const attributesToRender = transformItems(
            results.renderingContent?.facetOrdering?.facets?.order ?? [],
            { results }
          );
```


then a bit later: `attributesToRender` in the return

Reply:
good for me!